lib/server.js

```diff
@@ -8,6 +8,8 @@ const spawn = require('child_process').spawn
 const tmp = require('tmp')
 const fs = require('fs')
 const path = require('path')
+const {stringify} = require('flatted/cjs')
```

Comment:
Would you mind to wrap this dependency into `JSONUtils` and add unit tests for it, something like we have with [sha1 calculation](https://github.com/karma-runner/karma/blob/ad820a1d0998f0a81eb1d231e3f9b034d8ff21c6/test/unit/utils/crypto-utils.spec.js#L5-L9) ?

I like this kind of wrapping because of two reasons:
- dependencies sometimes change theirs APIs and behaviour - with wrapper covered by unit tests we're able to figure it out very early
- it's easier to remove dependency when it's used in single place, and it's behaviour is covered by unit tests.

Reply:
done